Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.mm

```diff
@@ -448,6 +514,146 @@ static bool isZoomSupported(const Vector<VideoPreset>& presets)
     return *m_capabilities;
 }
 
+AVCapturePhotoOutput* AVVideoCaptureSource::photoOutput()
+{
+    assertIsCurrent(RunLoop::main());
+
+    if (!m_photoOutput) {
+        m_photoOutput = adoptNS([PAL::allocAVCapturePhotoOutputInstance() init]);
+
+        if (![session() canAddOutput:m_photoOutput.get()]) {
+            ERROR_LOG_IF(loggerPtr(), LOGIDENTIFIER, "unable to add photo output");
+            return nullptr;
+        }
+        [session() addOutput:m_photoOutput.get()];
```

Comment:
Should we remove the output and destroy m_photoOutput when we are finished with the photo request ?
Or does it have not side effect to keep it even if we only use it once?